core/data_loader.py

```
@@ -203,80 +245,110 @@ def load_customers(self, csv_path='customer_data.csv', force_refresh=False):
+
         # Load customers from CSV
         customers_df = self.load_customers_from_csv()
-        
+
         # Enrich customer data with car models
         if not customers_df.empty and not inventory_df.empty:
             customers_df = self.enrich_customer_models(customers_df, inventory_df)
-        
+
         logger.info("✅ Data loading complete!")
         logger.info(f"📊 Final data summary:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)